streams/src/test/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImplTest.java

```diff
@@ -75,16 +75,19 @@ public void before() {
         groupedStream = stream.groupByKey(Serialized.with(Serdes.String(), Serdes.String()));
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
nit: Since a large majority of the tests need `@SuppressWarnings("deprecation")` why not put the annotation at the class level?

Reply:
Maybe... Had the same though but was not sure. \cc @guozhangwang @dguy

Reply:
I'd prefer having annotations on the function level to keep it finer grained. It helps other developers to look into these in the future.